app/src/main/java/app/revanced/integrations/youtube/swipecontrols/SwipeControlsConfigurationProvider.kt

```diff
@@ -104,5 +104,10 @@ class SwipeControlsConfigurationProvider(
     val shouldSaveAndRestoreBrightness: Boolean
         get() = Settings.SWIPE_SAVE_AND_RESTORE_BRIGHTNESS.get()
 
+    /**
+     * should the auto-brightness be enabled at the lowest value of the brightness gesture
+     */
+    val shouldEnableAutoBrightness: Boolean
```

Comment:
Maybe rename this to shouldEnableLowestValueAutoBrightness (including the settings property and key)